examples/spark2/src/main/scala/org/apache/carbondata/examples/PreAggregateDataMapExample.scala

```diff
@@ -51,6 +51,7 @@ object PreAggregateTableExample {
        LOAD DATA LOCAL INPATH '$testData' into table mainTable
        """)
 
+    // create datamaps of pre-aggregate
```

Comment:
better change to `create pre-aggregate table by datamap`

Reply:
accept,fixed it.